package.json

```diff
@@ -194,12 +195,13 @@
     "clean:public": "npm run -s rm -- public .css .css.map .js .js.map",
     "predashboard": "npm run -s install-if-deps-outdated",
     "dashboard": "cross-env DASHBOARD=1 webpack-dashboard -- npm start",
-    "distclean": "npm run -s clean && npm run -s rm -- node_modules",
+    "distclean": "npm run -s clean && lerna clean --yes && npm run -s rm -- node_modules",
```

Comment:
This is problematic as `lerna` is not global and nothing guarantees it will be present.